wwr/about.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="A Rafting Company">
+    <title>White Water Rafting</title>
```

Comment:
Please fix the title according to the course audio tool:

![image](https://github.com/user-attachments/assets/2d9773ab-4996-4713-8971-a65fcf99ddce)

You should add "About Us" along with the name of the rafting company in the title, like this:

```html
<title>About Us | Rapids</title>
```